packages/xo-server/src/xo-mixins/cloud-configs.js

```diff
@@ -38,7 +39,10 @@ export default class {
       app.addConfigManager(
         'cloudConfigs',
         () => db.get(),
-        cloudConfigs => db.update(cloudConfigs)
+        cloudConfigs =>
+          Promise.all(
+            mapToArray(cloudConfigs, cloudConfig => db.update(cloudConfig))
+          )
```

Comment:
Can you explain the issue?

Reply:
My bad db.update support a collection. I'll close this PR.